src/Input/ParamAwareInput.php

```diff
@@ -39,31 +37,31 @@ public function getParameterOption($values, $default = false, bool $onlyParams =
     }
 
     /**
-     * @return null|string|string[]
+     * @return mixed
```

Comment:
Do these really return `mixed`? :thinking:

Reply:
Sadly, it does.
https://github.com/symfony/console/blob/8b1008344647462ae6ec57559da166c2bfa5e16a/Input/InputInterface.php#L58

Reply:
😢